### PR TITLE
Zstd  compression support (from jbj)

### PR DESCRIPTION
**Pull request by proyvind**
Here's #256 turned into a PR, it depends on #282

### CODE REVIEW

**Comment by ffesti** [comment]
The code in rpm_check_lib.m4 and acinclude.m4 look pretty much the same. I kinda doubt we want or even need the code twice. PLease remove the file that is not needed.

**Comment by Conan-Kudo** [comment]
My feeling here is that we don't need `acinclude.m4` file, as it just has some extra fluff at the top. `rpm_check_lib.m4` looks to be sufficient, since configure.ac seems to already be configured to pull in the `m4` directory.

File: m4/acinclude.m4 [hunk 1]
```diff
@@ -0,0 +1,537 @@
+dnl ##
+dnl ##  acinclude.m4 -- manually provided local Autoconf macros
+dnl ##
+
+dnl ##
+dnl ##  NAME:
+dnl ##    AC_MSG_TITLE -- Display a configuration title
```
Comment (ignatenkobrain) [inline m4/acinclude.m4]: such things should go to autoconf (or autoconf-archive).

File: m4/acinclude.m4 [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+dnl ##    AC_MSG_TITLE(<name>, <version>)
+dnl ##
+
+AC_DEFUN([AC_MSG_TITLE],[
+    _AS_ECHO([Configuring $1, Version $2])
+])
+
+dnl ##
+dnl ##  NAME:
+dnl ##    AC_MSG_HEADER -- Display a configuration header
```
Comment (ignatenkobrain) [inline m4/acinclude.m4]: such things should go to autoconf (or autoconf-archive).

File: m4/acinclude.m4 [hunk 3]
```diff
@@ -0,0 +1,537 @@
...
+
+AC_DEFUN([AC_MSG_VERBOSE], [
+    if test ".$verbose" = .yes; then
+        _AS_ECHO([$1])
+    fi
+])
+
+dnl ##
+dnl ##  NAME:
+dnl ##    RPM_CHECK_LIB -- Check for third-party libraries
```
Comment (ignatenkobrain) [inline m4/acinclude.m4]: and such things as well... personally I think we should just stick to pkgconfig and do not care about anything else. pkgconfig is standard de-facto.

**Comment by ignatenkobrain** [comment]
superseded by #295

**Pull request closed without merging**